### PR TITLE
add Pod Manager References

### PR DESCRIPTION
**Pull request by rhatdan**
Just to clarify where the name came from. We also now manage
volumes so add that to the description.

Signed-off-by: Daniel J Walsh <dwalsh@redhat.com>

### CODE REVIEW

**Comment by rhatdan** [comment]
@TomSweeneyRedHat @vrothberg PTAL

File: README.md [hunk 1]
```diff
@@ -3,8 +3,7 @@
 # Library and tool for running OCI-based containers in Pods
 
 Libpod provides a library for applications looking to use the Container Pod concept,
-popularized by Kubernetes.  libpod also contains the `podman` tool, for managing
-Pods, Containers, and Container Images.
+popularized by Kubernetes.  libpod also contains the Pod Manager tool `(podman)`, Podman manages Pods, Containers, and Container Images, and Container Volumes.
```
Comment (vrothberg) [inline README.md]: `Podman` vs. `podman`, `libpod` vs. `Libpod`.

I think that `Podman manages Pods [...]` is a new sentence.

**Comment by vrothberg** [comment]
CI has a hiccup atm.

**Comment by mheon** [comment]
LGTM

**Comment by rhatdan** [comment]
Could someone give this a /lgtm

**Pull request merged**